plugins/public_suffix_list.dat.license

```diff
@@ -0,0 +1,2 @@
+SPDX-License-Identifier: MPL-2.0
+SPDX-FileCopyrightText: The Public Suffix List contributors and maintainers
```

Comment:
This one is tricky, as I don't know what to put in here. There is no oficial copyright information for the PSL (and that's not because nobody tried to find out: https://github.com/publicsuffix/list/issues/1121). @gotmax23 what do you think, is the above OK / "correct enough"?

Reply:
IANAL, but something like that or maybe "The Public Suffix List Authors" should suffice.

Reply:
```suggestion
SPDX-FileCopyrightText: The Public Suffix List Authors
```

Reply:
How about "The Public Suffix List Copyright Holders"? ;-) Just joking, your suggestion sounds good!